src/components/features/SaleBox/SaleBox.js

```diff
@@ -18,7 +18,7 @@ const SaleBox = sale_img => {
             <p className={styles.text_price}>-20%</p>
           </div>
           <div className={styles.right_inner}>
-            <div>
+            <div className={styles.top_inner}>
```

Comment:
słowa w nazwie klasy CSS rozdzielamy za pomocą `-`

Reply:
Wybacz nie wiem co jest powyżej komentarza mam pustą przestrzeń. 
Co do nazwa, chciałem tak zrobić ale Prettier zmienia mi ustawienia, dlatego wykorzystałem `_`. Czy to błąd?

Reply:
Wybacz, poprawiłem według instrukcji, ale przy commicie dostałem błąd. Wyłączyłem `Prettier`, ale to nie pomogło. Pozostawiłem jak było, cz to problem?

Reply:
a no tak, mój błąd, tak musi zostać

Reply:
tylko teraz widzę że w pliku scss jest `promo-text` a w komponencie `promo_text` - to nie zadziała

Reply:
Było więcej błędów, to już poprawione.